File: c#/rest-api/rest-api/Models/Person.cs
using System;
namespace rest_api.Models
{
    public class Person
    {
        String name;
        int age;

        public String Name
        {
            get { return name; }
            set { name = value; }
        }

        public int Age
        {
            get { return age; }
            set { age = value; }
        }
    }
}

File: kotlin-slow-query-detector/src/main/kotlin/com/codegik/hibernate/domain/Person.kt
package com.codegik.hibernate.domain

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import java.util.UUID


@Entity
data class Person(
    @Id
    val id: UUID,

    @Column
    val name: String,

    @Column
    val age: Int,
)